src/Main.cpp

```diff
@@ -96,6 +98,20 @@ int main(int argc, char** argv)
         }
         std::cout << wp_device.getConfig() << std::endl;
     }
+    else if (cmd == "set-id") {
+        if (argc != 7)
+        {
+            std::cerr << "wrong number of arguments" << std::endl;
+        }
+        power_whisperpower::Device device(device_group, device_id);
+        waitResult(*can_device, device, device.querySerialNumber());
+        uint32_t serial_number = device.getSerialNumber();
+        int device_new_id(std::stoi(argv[6]));
+        waitResult(*can_device, device, device.querySetId(0x600, serial_number));
```

Comment:
This makes no sense ... the new ID is not given anywhere in the package.

Reply:
I agree with you, but it what I understood about 
> we verified in live that we should send the set id query to can id 0x600 (broadcast)
@wvmcastro can you explain how did you test it?